### PR TITLE
add a quick docker-compose file

### PR DESCRIPTION
**Pull request by ayman2nov**
this is a quick runner in the local using docker hub images

### CODE REVIEW

File: docker-compose-quick.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - bundle
+    - exec
+    - rails
+    - server
+    - -b
+    - 0.0.0.0
+    ports:
+    - 3000:3000
+    labels:
+      io.rancher.container.pull_image: always
```
Comment (ayman2nov) [inline docker-compose-quick.yml:37]: Can remove Rancher labels

File: docker-compose-quick.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  redis:
+    image: redis:3.2-alpine
+    stdin_open: true
+    tty: true
+    labels:
+      io.rancher.container.pull_image: always
```
Comment (ayman2nov) [inline docker-compose-quick.yml:43]: Can remove Rancher labels

File: docker-compose-quick.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  postgres:
+    image: postgres:9.5
+    environment:
+      POSTGRES_PASSWORD: postgres
+    stdin_open: true
+    tty: true
+    labels:
+      io.rancher.container.pull_image: always
```
Comment (ayman2nov) [inline docker-compose-quick.yml:51]: Can remove Rancher labels

File: docker-compose-quick.yml (hunk 1 above)
Comment (michelson) [reply docker-compose-quick.yml:37]: Hi , note that you are adding to the source code your secrets , look at secret_token & secret_key_base

Comment (ayman2nov) [reply docker-compose-quick.yml:37]: Thank you, 
These are examples, not my Secrets

**Comment by michelson** [comment]
Hi @ayman2nov , thanks for this-

is there any documentation on how should I run this to test it ?

**Pull request closed without merging**